web/src/components/DandisetsPage.vue

```diff
@@ -39,33 +40,53 @@
           </v-list-item>
         </v-list>
       </v-menu>
-      <div class="mx-6">
-        Sort By:
-      </div>
-      <v-chip-group
-        :model-value="sortOption"
-        selected-class="white text-light-blue bg-white"
-        dark
-        mandatory
-        style="min-width: 25%"
+      <DandisetSearchField class="flex-grow-1 mr-2" />
+      <v-btn-group
+        variant="flat"
+        density="comfortable"
+        divided
+        class="btn-group--sort-options"
       >
-        <v-chip
-          v-for="(option, i) in sortingOptions"
-          :key="option.name"
-          @click="changeSort(i)"
+        <v-btn class="btn--sort-option">
+          <v-icon>mdi-sort-variant</v-icon>
+          <v-tooltip
+            location="top"
+            activator="parent"
+          >
+            <span>Sorting Options</span>
+          </v-tooltip>
+          <v-menu activator="parent">
+            <v-list>
+              <v-list-item
+                v-for="(option, i) in sortingOptions"
+                :key="option.name"
+                :value="option"
+                :append-icon="option.name === sortingOptions[sortOption].name ? 'mdi-check' : ''"
```

Comment:
This is more subjective than the other two points, but you might consider using the [`append` slot](https://vuetifyjs.com/en/api/v-list-item/#slots-append) here instead of the `append-icon` prop. With that, you could avoid inlining a ternary expression into the template and instead just conditionally render the slot with `v-if`.

Reply:
Added to b2e7fee